### PR TITLE
feat: wishlist and cart ui

### LINKED ISSUE #30: update: wishlist and cart ui

### PR DESCRIPTION
**Pull request by Bhakti199**
here I updated wishlist and cart page ui. #30

### CODE REVIEW

File: src/Components/CartCards/ProductCardForCart.jsx [hunk 1]
```diff
@@ -2,42 +2,53 @@ import React from "react";
...
 import toast from "react-hot-toast";
-import { FaRupeeSign } from "react-icons/fa";
 export const ProductCardForCart = () => {
   const { state, dispatch } = useFilter();
-  const { cartList } = state;
+  const { cartList, wishList } = state;
   return (
     <>
       {cartList &&
         cartList.map((item) => (
```
Comment (t007rushi) [inline src/Components/CartCards/ProductCardForCart.jsx:13]: destructure here as ```{cartItem:{id,img,...}}```

File: src/Pages/CartPage/CartPage.jsx [hunk 2]
```diff
@@ -2,15 +2,23 @@ import React from "react";
...
-
+    <div className="cart-page-container bg-color">
+      <h2 className="cart-page-title">CART</h2>
+      {cartList.length === 0 && (
+        <div className="cart-empty-msg">
+          <span className="cart-empty-msg-title">
+            YOUR SHOPPING CART IS EMPTY!
+          </span>
+          <Link to="/product-listing">
+            <button className="cart-msg-btn">Shop Now</button>
```
Comment (t007rushi) [inline src/Pages/CartPage/CartPage.jsx:18]: is a button tag needed? here as you can give this className to link itself

File: src/Pages/WishListPage/WishListPage.jsx [hunk 3]
```diff
@@ -1,85 +1,102 @@
...
 import { useFilter } from "../../Context/index";
-import { BsFillStarFill, BsFillHeartFill } from "react-icons/bs";
+import { BsFillStarFill, BsFillHeartFill, BsCart2 } from "react-icons/bs";
 import { FaRupeeSign } from "react-icons/fa";
 import toast from "react-hot-toast";
+import { Link } from "react-router-dom";
 export const WishListPage = () => {
   const { state, dispatch } = useFilter();
   const { wishList } = state;
   return (
```
Comment (t007rushi) [inline src/Pages/WishListPage/WishListPage.jsx:12]: have some spacing btwn

File: src/Pages/WishListPage/WishListPage.jsx [hunk 4]
```diff
@@ -1,85 +1,102 @@
...
-                          }),
-                            toast("removed from wishlist", { icon: "❌" });
-                        }}
-                      />
-                    </button>
-                  </div>
-                  <div className="vertical-card-content">
-                    <div className="flex">
-                      <h3 className="vertical-card-content-one margin-top-bottom-zero margin-auto">
+              <div className="wishlist-card">
```
Comment (t007rushi) [inline src/Pages/WishListPage/WishListPage.jsx:19]: good practice to have key={} immediate tag

**Review by t007rushi — APPROVED** [review]

**Review by tushartiwari7 — APPROVED** [review]
Nice ui
Implement the above mentioned changes and you are good to merge.

**Pull request merged**